vsts/pipelines/templates/_releaseJobTemplate.yml

```diff
@@ -78,5 +78,9 @@ jobs:
       repositoryName: Microsoft/Oryx
       tagSource: manual
       tag: '$(RELEASE_TAG_NAME)'
-      assets: $(Build.SourcesDirectory)/artifacts/oryx_linux_x64.tar.gz
+      assets: |
+          $(Build.SourcesDirectory)/artifacts/oryx_linux_x64.tar.gz
+          $(Build.SourcesDirectory)/artifacts/Microsoft.Oryx.Detector.0.2.0.nupkg
+          $(Build.SourcesDirectory)/artifacts/Microsoft.Oryx.Detector.Cli.0.2.0.nupkg
```

Comment:
So for the detector and detector.cli that later we add on, we need to match these two names here?

Reply:
After our discussion few days ago, for now I am creating packages based on existing BuildScriptGenerator and it's CLI